packages/extract-react-types/src/index.js

```diff
@@ -1217,31 +1229,11 @@ converters.TSTypeQuery = (path, context): K.TypeQuery => ({
   exprName: convert(path.get('exprName'), { ...context, mode: 'value' })
 });
 
-function convertMethodCall(path, context): K.Func {
-  const parameters = path.get('parameters').map(p => convertParameter(p, context));
-  const returnType = convert(path.get('typeAnnotation'), context);
-
-  return {
-    kind: 'function',
-    returnType,
-    parameters
-  };
-}
-
-function mapComment(comment) {
-  return {
-    type: comment.type === 'CommentLine' ? 'commentLine' : 'commentBlock',
-    value: normalizeComment(comment),
-    raw: comment.value
-  };
-}
-
 function attachCommentProperty(source, dest, name) {
```

Comment:
would be cool and arguably more testable if this function was pure, and simply returned its result rather than manipulating params? Would also make the corresponding call on L1302 chainable.

Reply:
Good feedback man, yeah I did feel a bit weird about this function too. Might see if I can improve it :D